### ISSUE TITLE
Coding: Menu for Emojis

### ISSUE THREAD

**Issue by tuckplace** [cs-utulsa/Encrypted-Chat-Service#81]
Please make sure you read over the [emojis](https://github.com/cs-utulsa/Encrypted-Chat-Service/wiki/Feature-Release-4:-Emojis) page before starting this issue.

In order for a user to select and pick an emoji to send, a menu is required. On our main chat screen next to the file transfer button, a new button needs to be added for emojis. When a user clicks this button, a menu should open at the bottom.

This menu should be part of our main window, not a new one. On this menu, there should be each supported emoji and a scroll bar. Each emoji must be clickable. When an emoji is selected, it should be added to the chat bar on the message. There should then be a button to hide the emojis menu. 

- [x] Button on main chat screen for emojis
- [x] Menu for emojis is displayed when button is clicked and is part of the main window.
- [x] While the emoji menu is displayed, a button is visible to hide the emoji menu. 
- [x] The emoji menu contains a scroll bar and buttons for each emoji
- [x] When an emoji button is clicked, the ID :ID123: is added to the message being written. (Another issue exists to properly display emojis in the chat bar)
- [x] Code is clean and commented.

**Comment by piangno** [comment]
7 hours
